README.md

```diff
@@ -1,7 +1,95 @@
 o-cookie-message [![Circle CI](https://circleci.com/gh/Financial-Times/o-cookie-message/tree/master.svg?style=svg)](https://circleci.com/gh/Financial-Times/o-cookie-message/tree/master)
 =================
 
-Origami Cookie Message.
+The cookie message and behaviour approved by the FT's legal team.
+All FT websites must have a cookie message. Using o-cookie-message will ensure your site is compliant.
+
+- [Usage](#usage)
+	- [Markup](#markup)
+	- [JavaScript](#javascript)
+	- [Sass](#sass)
+	- [Customising o-cookie-message HTML](customising-o-cookie-message-html)
+- [Migration guide](#migration-guide)
+- [Contact](#contact)
+- [Troubleshooting](#troubleshooting)
+
+## Usage
+
+### Markup
+
+Use the following HTML to get a full width banner with the legal approved cookie message.
+```
+<div data-o-component="o-cookie-message" class='o-cookie-message o-cookie-message--banner-centric'>
+</div>
+```
+
+### JavaScript
+
+No code will run automatically unless you are using the Build Service. An `o-cookie-message` object must be constructed.
```

Comment:
They can also fire the `O-DOMContentLoaded` event as `o-cookie-message` should be listening for it.